myaws/ssm_parameter_env.go

```diff
@@ -35,7 +35,7 @@ func (client *Client) SSMParameterEnv(options SSMParameterEnvOptions) error {
 		output = append(output, formatSSMParameterAsEnv(parameter, options.Name, options.DockerFormat))
 	}
 
-	fmt.Fprintf(client.stdout, strings.Join(output[:], " "))
+	fmt.Print(strings.Join(output[:], " "))
```

Comment:
Can we use `fmt.Fprint` instead of `fmt.Print`?
Current implementation does not have tests, but using `fmt.Fprint` is useful for testing.

Reply:
Thanks!
I fixed it.